src/mmw/js/src/compare/models.js

```diff
@@ -208,6 +208,25 @@ var WindowModel = Backbone.Model.extend({
         scenarios: null, // ScenariosCollection
         tabs: null,  // TabsCollection
         visibleScenarioIndex: 0, // Index of the first visible scenario
+        polling: false,  // If any results are polling
+    },
+
+    initialize: function() {
+        var setPolling = _.bind(this.setPolling, this);
+
+        this.get('scenarios').forEach(function(scenario) {
+            scenario.get('results').on('change', setPolling);
+        });
+    },
+
+    setPolling: function() {
+        var getPolling = function(scenario) {
+                return scenario.get('results').pluck('polling');
+            },
+            scenarios = this.get('scenarios'),
+            polling = _.flatten(scenarios.map(getPolling)).some(_.identity);
```

Comment:
Probably fine to leave but this could also use the lodash/underscore version of `some`?